src/style.css

```diff
@@ -0,0 +1,145 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
```

Comment:
you should connect 3 fonts for 3 different font-weights, and they should have same font-family